src/OrchardCore.Modules/OrchardCore.Forms/Controllers/ValidationRuleController.cs

```diff
@@ -0,0 +1,87 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Localization;
+using OrchardCore.ContentManagement;
+using OrchardCore.Forms.Helpers;
+using OrchardCore.Forms.Models;
+using OrchardCore.Forms.Services;
+using OrchardCore.Forms.Services.Models;
+
+namespace OrchardCore.Forms.Controllers
+{
+    [Route("api/validationApi")]
+    [Authorize(AuthenticationSchemes = "Api"), IgnoreAntiforgeryToken, AllowAnonymous]
+    [ApiController]
+    public class ValidationApiController : Controller
+    {
+        private readonly IAuthorizationService _authorizationService;
+        private readonly ValidationRuleHelpers _validationRuleHelpers;
+        private readonly IStringLocalizer S;
+        public ValidationApiController(
+            IAuthorizationService authorizationService,
+            ValidationRuleHelpers validationRuleHelpers,
+            IStringLocalizer<ValidationApiController> stringLocalizer)
+        {
+            _authorizationService = authorizationService;
+            _validationRuleHelpers = validationRuleHelpers;
+            S = stringLocalizer;
+        }
+        [HttpGet]
+        [Route("ValidateFormByRule")]
+        public async Task<IActionResult> ValidateFormByRule(string contentItemId, string formName, string formValue)
+        {
+            if (String.IsNullOrWhiteSpace(contentItemId) || String.IsNullOrWhiteSpace(formName))
+            {
+                return BadRequest(S["contentItemId, formName are required parameters"]);
+            }
+
+            //if (!await _authorizationService.AuthorizeAsync(User, Permissions.ValidationRule))
```

Comment:
No need for a permission to validate something, and the Permission class should then be removed too.